CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# Contributing
+
+## Installation for Local Development
+
+[Poetry][poetry-link] is used to manage the python development environment. 
+
+A simple way to create an environment with the desired version of python and poetry is to use [conda][conda-link].  E.g.:
+
+```bash
+conda create -n fgpyo -c conda-forge "python>=3.9" poetry
+conda activate fgpyo
+
+# --all-extras is required to install `mkdocs` and associated dependencies,
+# which are required for development 
+poetry install --all-extras
+```
+
+[rtd-link]:    http://fgpyo.readthedocs.org/en/stable
+[poetry-link]: https://github.com/python-poetry/poetry
+[conda-link]:  https://docs.conda.io/en/latest/miniconda.html
+
+## Primary Development Commands
+
+To check and resolve linting issues in the codebase, run:
+
+```console
+poetry run ruff check --fix
+```
+
+To check and resolve formatting issues in the codebase, run:
+
+```console
+poetry run ruff format
+```
+
+To check the unit tests in the codebase, run:
+
+```console
+poetry run pytest
+```
+
+To check the typing in the codebase, run:
+
+```console
+poetry run mypy
+```
+
+To generate a code coverage report after testing locally, run:
```

Comment:
The pytest command above (mirrored from GitHub Actions workflow) now makes the HTML report by default.

```suggestion
To re-generate a code coverage report after testing locally, run:
```